src/models/dune_json_formats.rs

```diff
@@ -39,10 +40,14 @@ mod tests {
                     {
                         "data": {
                             "cowswap_usd_volume": 474.26231998787733,
-                            "day": "2021-05-05",
+                            "day": "2021-10-18T00:00:00+00:00",
```

Comment:
Any reason for this change? Did you remove a `DATE()` method in a query somewhere?

Reply:
Yes, maybe. I will reinsert it with another PR